src/tablesort.js

```diff
@@ -140,6 +141,10 @@
           sortDir = 'sort-down';
         } else if (header.classList.contains('sort-down')) {
```

Comment:
Perhaps instead of relying on the class for sort ordering it should solely use the data attribute?